### PR TITLE
Update Dockerfile to install tini from base image

### PR DESCRIPTION
**Pull request by msherif1234**
Signed-off-by: Mohamed Mahmoud <mmahmoud@redhat.com>

### CODE REVIEW

File: Dockerfile.openshift [hunk 1]
```diff
@@ -14,25 +14,22 @@ RUN INSTALL_PKGS=" \
...
 # variables to direct operation and configure ovn
 ADD frr.sh /root/
 ADD daemons /etc/frr
 ADD frr.conf /etc/frr
 ADD vtysh.conf /etc/frr
 
-RUN chown frr:frr /etc/frr/daemons /etc/frr/frr.conf 
+RUN chown frr:frr /etc/frr/daemons /etc/frr/frr.conf
+RUN ln -s /usr/bin/tini /sbin/tini
+RUN chmod +x /sbin/tini
```
Comment (fedepaol) [inline Dockerfile.openshift:32]: do we need this?

**Comment by fedepaol** [comment]
/hold
one last minute comment

File: Dockerfile.openshift (hunk 1 above)
Comment (msherif1234) [reply Dockerfile.openshift:32]: I am not sure when Idid it with u/s it needed it, we can try w/o it and if it didnt work we can add it

Comment (fedepaol) [reply Dockerfile.openshift:32]: :+1:

Comment (msherif1234) [reply Dockerfile.openshift:32]: give me a min checking quickly

**Pull request merged**